config/locales/fr.yml

```diff
@@ -150,24 +154,24 @@ fr:
       web: Web
     action_logs:
       actions:
-        assigned_to_self_report: "%{name} s'est assigné le signalement de %{target} à eux-même"
-        change_email_user: "%{name} a modifié l'adresse de courriel de l'utilisateur %{target}"
-        confirm_user: "%{name} adresse courriel confirmée de l'utilisateur %{target}"
-        create_custom_emoji: "%{name} a importé de nouveaux emoji %{target}"
+        assigned_to_self_report: "%{name} s’est assigné le signalement de %{target} à eux-même"
+        change_email_user: "%{name} a modifié l’adresse de courriel de l’utilisateur·rice %{target}"
+        confirm_user: "%{name} adresse courriel confirmée de l’utilisateur·ice %{target}"
+        create_custom_emoji: "%{name} a importé de nouveaux émoji %{target}"
```

Comment:
émojis (if the word is Frenchified, then it should also follow all the usual rules of French including plural rules)